big_tests/tests/ejabberdctl_SUITE.erl

```diff
@@ -907,10 +911,17 @@ simple_register(Config) ->
     Domain = ct:get_config({hosts, mim, domain}),
     {Name, Password} = {<<"tyler">>, <<"durden">>},
     %% when
-    {_, 0} = ejabberdctl("register", [Name, Domain, Password], Config),
+    {R1, 0} = ejabberdctl("registered_users", [Domain], Config),
+    {match, ResList1} = re:run(R1, "(.+)", [global]),
```

Comment:
Isn't it less obscure to just use `string:tokens`? :)

Reply:
Indeed